packages/trader/src/Modules/CFD/Containers/cfd-server-error-dialog.tsx

```diff
@@ -1,9 +1,18 @@
-import PropTypes from 'prop-types';
 import React from 'react';
 import { Dialog } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 
+type TCFDServerErrorDialog = {
```

Comment:
I suggest the following 😊:
1) adding Props to type name to specify that it's a props type, not a component's type;
2) adding `?` (undefined option) only to `error_type` that actually can be `undefined` (the rest of the props are initialized as defined values (''/false/() => void) in the stores they are taken from).
```
type TCFDServerErrorDialogProps = {
    clearCFDError: () => void;
    disableApp: () => void;
    enableApp: () => void;
    error_message: string;
    error_type?: string;
    has_cfd_error: boolean;
    is_cfd_success_dialog_enabled: boolean;
};
```